### PR TITLE
Sync CRD spec with bootstrap project

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Hi there 👋 

This PR syncs the CRD spec with the bootstrap project I pushed yesterday.

There are a couple of changes to the CRDs themselves and a bunch on wording, but nothing major. **Please let me know what you think**.

Once this is merged, I'll open issues to discuss some points of the spec.

Cheers 🍻

### CODE REVIEW

File: Spec.md [hunk 1]
```diff
@@ -25,13 +26,13 @@ status: draft
 
 ## Summary
 
-This is a proposal to standardize the `ExternalSecret` CRD in an combined effort through all projects that deal with syncing external secrets. This proposal aims to do find the _common denominator_ for all users of `ExternalSecrets`.
+This is a proposal to standardize the External Secret Kubernetes operator CRDs in an combined effort through all projects that deal with syncing external secrets. This proposal aims to do find the _common denominator_ for all users of an External Secrets project.
```
Comment (moolen) [inline Spec.md]:
```suggestion
This is a proposal to standardize the External Secrets Kubernetes operator CRDs in an combined effort through all projects that deal with syncing external secrets. This proposal aims to do find the _common denominator_ for all users of an External Secrets project.
```

File: Spec.md [hunk 2]
```diff
@@ -91,40 +93,47 @@ These providers are relevant for the project:
...
-  # the amount of time before the values will be read again from the provider
-  # may be set to zero to fetch and create it once
+  # SecretStoreRef defines which SecretStore to fetch the ExternalSecret data
+  secretStoreRef:
+    name: secret-store-name
+    kind: SecretStore  # or ClusterSecretStore
+
+	# RefreshInterval is the amount of time before the values reading again from the SecretStore provider
+	# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h" (from time.ParseDuration)
+	# May be set to zero to fetch and create it once
```
Comment (moolen) [inline Spec.md]: Here are a bunch of tabs. These should be whitespaces, right?

File: Spec.md [hunk 3]
```diff
@@ -198,31 +204,43 @@ status:
...
-### External Secret Store
+### Secret Store
 
 The store configuration in an `ExternalSecret` may contain a lot of redundancy, this can be factored out into its own CRD.
-These stores are defined in a particular namespace using `SecretStore` **or** globally with `GlobalSecretStore`.
+These stores are defined in a particular namespace using `SecretStore` **or** globally with `ClusterSecretStore`.
 
 ```yaml
-apiVerson: external-secrets.k8s.io/v1alpha1
+apiVerson: external-secrets.x-k8s.io/v1alpha1
```
Comment (moolen) [inline Spec.md]: AFAIK `x-k8s.io` is reserved for SIG sponsored projects. I'll try to find out what that actually means (couldn't find much in the k8s docs). This is not a blocker for this PR tho. We'll change that later anyway

**Review by moolen — CHANGES_REQUESTED** [review]
I found a typo and tabs. As you have proposed via slack it would make sense to remove the `Kubernetes` from `Kubernetes External Secrets` and just stick with `external-secrets` for the name.

File: Spec.md (hunk 3 above)
Comment (jonatasbaldin) [reply Spec.md]: Agreed. What do you think about only having one subdomain, `external-secrets.io`, like the examples below?

```yaml
apiVersion: external-secrets.io/v1alpha1
kind: ExternalSecret
metadata:
  name: externalsecret-sample
spec:
  # Add fields here
  foo: bar
```

```yaml
apiVersion: external-secrets.io/v1alpha1
kind: SecretStore
metadata:
  name: secretstore-sample
spec:
  # Add fields here
  foo: bar
```

Comment (jonatasbaldin) [reply Spec.md]: @knelasevero

**Comment by jonatasbaldin** [comment]
@moolen did the changes. I changed the KES term for ESO, whenever referring to an Operator instance.

**Review by moolen — APPROVED** [review]
LGTM, thanks!

File: Spec.md (hunk 3 above)
Comment (moolen) [reply Spec.md]: yeah, that's good!

**Comment by jonatasbaldin** [comment]
Perfect, merging! Also created [this PR](https://github.com/external-secrets/external-secrets/pull/4) to fix the subdomain change on the code base.

**Pull request merged**